app/lib/search/index_simple.dart

```diff
@@ -73,11 +81,8 @@ class SimplePackageIndex implements PackageIndex {
       addAll(_nameIndex.search(query.packagePrefix), 0.8);
     }
 
-    final Map<String, double> popularityScores = new Map.fromIterable(
-      total.keys,
-      value: (String url) => _documents[url].popularity * 100,
-    );
-    addAll(popularityScores, 0.05);
+    addAll(getHealthScore(total.keys), 0.05);
+    addAll(getPopularityScore(total.keys), 0.05);
```

Comment:
I agree, but my quick experiments showed that it is easy to screw up the balance, e.g. the test case for "chrome socket" will put the `chrome_net` package as last if I increase the popularity arbitrarily. Let's go with the current low value for now, and I'll experiment with the fine-tuning in a separate PR (with probably more test cases).